src/containers/favorites-page/store/sagas.ts

```diff
@@ -33,8 +40,18 @@ const getTrackFavorites = createUserListProvider<Track>({
   getExistingEntity: getTrack,
   extractUserIDSubsetFromEntity: (track: Track) =>
     track.followee_saves.map(r => r.user_id),
-  fetchAllUsersForEntity: ({ limit, offset, entityId }) =>
-    AudiusBackend.getFavoritersForTrack({ limit, offset, trackId: entityId }),
+  fetchAllUsersForEntity: ({ limit, offset, entityId, currentUserId }) => {
+    const trackId = encodeHashId(entityId)!
+    const encodedUserId = currentUserId
+      ? encodeHashId(currentUserId) ?? undefined
+      : undefined
+    return apiClient.getTrackFavoriteUsers({
```

Comment:
nit, I think we should put the encoded hash id calls inside apiClient.getTrackFavoriteUsers. I think michael did this

Reply:
I don't think I did this actually, but I agree w you. Less leaky abstraction if we can keep all the opaque IDs contained in this client